app/modules/collaborations/resources.py

```diff
@@ -97,17 +97,18 @@ def post(self, args):
         users = [current_user, other_user]
 
         if current_user.is_user_manager:
-            second_user_guid = req.get('second_user_guid')
-            second_user = User.query.get(second_user_guid)
-            if not second_user:
-                abort(400, f'User with guid {second_user_guid} not found')
-            if not second_user.is_active:
-                abort(400, f'User with guid {second_user_guid} is not active')
 
-            users = [other_user, second_user]
+            second_user_guid = req.get('second_user_guid')
+            if second_user_guid:
+                second_user = User.query.get(second_user_guid)
+                if second_user:
+                    if not second_user.is_active:
+                        abort(400, f'User with guid {second_user_guid} is not active')
```

Comment:
Good catch, I'd suggest changing the error to "Second user with guid" so that  this is a more meaningful response. 
Same comment on not found abort below